14/index.php

```diff
@@ -0,0 +1,174 @@
+<h3>What's Next?</h3>
+<ul>
+  <li>
+    Check the <a href="https://blog.keyman.com/2020/03/keyman-roadmap-march-2020/">Keyman Roadmap</a> for upcoming features.
+  </li>
+</ul>
+<br>
+
+<h3>14.0 Beta Feedback</h3>
+<ul>
+  <li>Please send feedback about Keyman 14 to the
+    <a href="https://community.software.sil.org/t/keyman-14-beta-feedback/4115">Keyman Community site</a> or submit bugs and feature requests to our
+    <a href="https://github.com/keymanapp/keyman/issues/new/choose">Issue Tracker</a></li>
+</ul>
+<br>
+
+<h1 class='red underline'>User Software</h1>
+
+<?php
+downloadSection('Keyman for Windows 14',   'windows',     'keyman-$version.exe', 'beta');
+?>
+
+<h3>What's New in Keyman for Windows 14?</h3>
+
+<ul>
+  <li>Renamed from <strong class="red">Keyman Desktop</strong> to <strong class="red">Keyman for Windows</strong></li>
+  <li>Handles major version upgrades from 10 onward</li>
+  <li>Add GUI for setting configuration options</li>
+</ul>
+
+
+
+<?php
+downloadSection('Keyman 14 for macOS',   'mac',     'keyman-$version.dmg', 'beta');
+?>
+
+<h3>What's New in Keyman 14 for macOS?</h3>
+
+<ul>
+  <li>Added icon for Keyman.app</li>
+</ul>
+
+
+
+<h2 id='linux' class='red underline'>Keyman 14 for Linux</h2>
+
+<h3>What's New in Keyman 14 for Linux?</h3>
+
+<ul>
+  <li>Open a .kmp file with Keyman Config</li>
+  <li>Now supports Ubuntu 20.10 (Groovy)</li>
+  <li>Improved UI</li>
+</ul>
+
+<li>Ubuntu, Wasta-Linux: Keyman for Linux can be installed via launchpad:</li>
+<blockquote><pre class='language-bash code'><code>sudo add-apt-repository ppa:keymanapp/keyman-nightly
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install keyman onboard</code></pre></blockquote>
+
+
+<?php
+downloadSection('Keyman for Android 14', 'android', 'keyman-$version.apk', 'beta');
+?>
+
+<p>You can also <a href=<?= $playstore_signup_link ?>>sign up</a> to access pre-release versions through Google Play.</p>
+<?= $playstoreTable ?>
+
+<h3>What's new in Keyman for Android 14?</h3>
+
+<ul>
+  <li>
+    Select a language during keyboard package installation<br>
+    <img alt='Select language during keyboard installation' src='select_language.png'>
+  </li>
+  <li>Consolidated install menus for installing keyboards</li>
+  <li>Add system globe action to show system keyboards</li>
+</ul>
+
+<h2 class="red underline">Keyman for iPhone and iPad 14</h2>
+<h3>Beta</h3>
+<p>Please register for the pre-release version through the links below for Keyman <a href=<?= $testflight_beta_link ?>>beta</a>
+  pre-releases on your iOS device. This will grant access to the app through <a  href="https://developer.apple.com/testflight/testers/">Apple's
+    Testflight app</a>, which facilitates direct installation on iOS devices.</p>
+
+<?= iosTestFlightTable(); ?>
+
+<h3>What's new in Keyman for iPhone and iPad 14?</h3>
+
+<ul>
+  <li>Select a language during keyboard package installation</li>
+  <li>Improved batching of keyboard and dictionary downloads</li>
+</ul>
```

Comment:
@jahorton, do you want anything from this list (you'll need to rewrite the titles from an end-user perspective)?

```
$ grep -E '^\* [a-z]+\(ios' HISTORY.md | grep feat
* feat(ios/engine): launches external links outside the app (#3889)
* feat(ios/engine): Utilizes packages' welcome pages (#3622)
* feat(ios/engine): go/package use for model pkg downloads (#3603)
* feat(ios/app): universal links for keyboard installation (#3466)
* feat(ios/engine): Associating package installer (#3458)
* feat(ios/app): Associating package installer use within existing install paths (#3465)
* feat(ios/engine): LanguagePickAssociator progress tracking, base integration with package lang picker (#3455)
* feat(ios/engine): Language-pick associator (#3451)
* feat(ios/engine): language selection during (file-based) package installation (#3416)
* feat(ios/engine) Package-installer layout optimizations (#3437)
* feat(ios): Start of engine and app internationalization (#2745)
* feat(ios/engine): download queue concurrency (#3395)
* feat(ios/engine): keyboard search core (#3384)
* feat(ios/engine): support for deferred chaining of lexical model downloads (#3401)
* feat(ios/engine): use of Keyman staging sites (#3405)
* feat(ios/engine): package-version query implementation (#3280)
* feat(ios/engine): adds getInstalledPackage(for:) (#3270)
* feat(ios/engine): KeymanPackage now parses its ID (#3234)
* feat(ios/engine): Generation of kmp.json for non-package resources (#3212)
* feat(ios/engine): embedded KeymanWeb engine Sentry use (#3218)
* feat(ios/engine): kmp.json - full deserialization (#3198)
* feat(ios/engine): HTTPDownloader unit testing (#3180)
* feat(ios): Add Crowdin CLI for iOS strings (#2905)
* feat(ios): compilations within Xcode now properly set version (#2775)
* feat(ios): Start of Sentry-based crash reporting (#2771)
* feat(ios): Improved Sentry integration (first pass) (#2782)
* feat(ios): KeymanEngine version migration unit tests (#2692)
* feat(ios): starts iOS unit testing (#2649)
* feat(ios): Adds keyboard-scale unit tests, fixes unknown-device bug (#2695)
* feat(ios): Deeplink into Keyman settings for system keyboard setup (#2548)
```

Reply:
```suggestion
<ul>
  <li>Select a language during keyboard package installation</li>
  <li>Improved batching of keyboard and dictionary downloads</li>
  <li>Improved correction suggestions (#3555)</li>
  <li>Match user input capital letters when offering suggestions (#3845)</li>
</ul>
```